android/build.gradle

```diff
@@ -45,6 +45,6 @@ repositories {
 
 dependencies {
     implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
-    def iidVersion = safeExtGet('googlePlayServicesIidVersion', safeExtGet('googlePlayServicesVersion', '17.0.0'))
-    implementation "com.google.android.gms:play-services-iid:$iidVersion"
+    def iidVersion = safeExtGet('iidVersion', safeExtGet('iidVersion', '19.0.1'))
+    implementation "com.google.firebase:firebase-iid:$iidVersion"
```

Comment:
did IID move completely to firebase? I'm using firebase heavily but hadn't noticed! Would explain the change above, but to make it non-breaking you have to chain the safeExtGet's so that it falls back to the existing one and doesn't break the installed userbase